wagtail/admin/tests/viewsets/test_model_viewset.py

```diff
@@ -1464,6 +1465,11 @@ def test_edit_form_rendered_with_panels(self):
             self.assertIsNotNone(rendered_heading)
             self.assertEqual(rendered_heading.text.strip(), expected_heading)
 
+        # Ensure modal-workflow.js is included, as it's needed by choosers
+        modal_workflow_js = versioned_static("wagtailadmin/js/modal-workflow.js")
+        modal_workflow_script = soup.select_one(f'script[src="{modal_workflow_js}"]')
+        self.assertIsNotNone(modal_workflow_script)
```

Comment:
I've added this assertion and restored the `assertTemplateUsed` lines in the tests for wagtailsites.

Reply:
Thanks for that. I don't seem to be getting CI failure notifications since this week for some reason. I appreciate you finishing this of.